deploy/scripts/setup_combine.py

```
@@ -184,15 +185,15 @@ def main() -> None:
             logging.debug(f"Installed charts: {installed_charts}")
 
-            # Set helm_action based on whether chart is already installed
-            helm_action = HelmAction.INSTALL
-            if chart in installed_charts:
-                if args.clean:
-                    # Delete existing chart if --clean specified
-                    delete_cmd = helm_cmd + [f"--namespace={chart_namespace}", "delete", chart]
-                    run_cmd(delete_cmd, print_cmd=not args.quiet, print_output=True)
-                else:
-                    helm_action = HelmAction.UPGRADE
+            namespace_cmd = helm_cmd + [f"--namespace={chart_namespace}"]
```

Ruff (0.15.6)

[warning] Consider `[*helm_cmd, f"--namespace={chart_namespace}"]` instead of concatenation

Replace with `[*helm_cmd, f"--namespace={chart_namespace}"]`

(RUF005)